.github/workflows/tools.yml

```diff
@@ -167,6 +167,14 @@ jobs:
               cat temp-output
               tail -n1 temp-output | grep "NEW_VERSION=" >> "$GITHUB_ENV" || true
               rm temp-output
+          - id: cjs-module-lexer
+            subsystem: deps
+            label: dependencies
+            run: |
+              ./tools/dep_updaters/update-cjs-module-lexer.sh > temp-output
+              cat temp-output
```

Comment:
Not a blocker, but I can't really say if you left this while debugging or it's actually needed. Just pointing this out for you to double check.

Reply:
for debugging purposes we use it in every job